### ISSUE TITLE
Slayer Tasks

### ISSUE THREAD

**Issue by BeeZZy03** [t8or/runelite-recommended-equipment#5]
can you add more slayer tasks?

**Comment by t8or** [comment]
@BeeZZy03 Of course! Do you have some Wiki URLs to the slayer activities you are referencing?

**Comment by Oridjinn1980** [comment]
I would think most tasks that have a "demi-boss" type monster that you can choose to kill. Two examples that I can think of would be [Tormented Demons](https://oldschool.runescape.wiki/w/Tormented_Demon/Strategies) (for Greater Demons) and [Lizardman Shamans](https://oldschool.runescape.wiki/w/Lizardman_shaman/Strategies) for Lizardmen.

**Comment by t8or** [comment]
@Oridjinn1980 Thank you for these examples! I have added them to the plugin, and they should be live in the next 5 or so minutes. @BeeZZy03 If you can think of any other URLs with gear sections for Slayer monsters, please post them here! Thanks.

**Comment by BeeZZy03** [comment]
Yeah I have an idea! I’ve found a lot of guides to NMZ but none for noobs who need to farm points. Would you post a recommended NMZ Point Farm under skill up or somewhere?

**Comment by Charliehall10** [comment]
is there a way to see reccomended inventory too perhaps?

**Comment by t8or** [comment]
> is there a way to see reccomended inventory too perhaps?
> 
> 

@Charliehall10 tbus is captured in a separate issue here [Add Recommended Inventory to plugin](https://github.com/t8or/runelite-recommended-equipment/issues/8)

**Comment by KVINTH** [comment]
can you add 

https://oldschool.runescape.wiki/w/Slayer_task/Custodian_stalker

**Comment by t8or** [comment]
> can you add 
> 
> https://oldschool.runescape.wiki/w/Slayer_task/Custodian_stalker

Added stalkers!! They should be in the plug-in now. Ty for URL 🙏

**Comment by Brandon1212b** [comment]
How much work is it to add every slayer strategy? For example, we're missing
- https://oldschool.runescape.wiki/w/Slayer_task/Nechryael
- https://oldschool.runescape.wiki/w/Dust_devil/Strategies
- https://oldschool.runescape.wiki/w/Slayer_task/Gargoyles
- plus the many others found here: https://oldschool.runescape.wiki/w/Slayer_task

This might not be your goal of the plugin, but this could make slayer really easy when it's paired with the slayer assistant plugin that already tells you where to go

**Comment by adamk33n3r** [comment]
> How much work is it to add every slayer strategy? 

It really depends on how the wiki is set up. We have a separate project that scrapes the wiki every week. It looks for the recommended equipment widget on the pages listed in this file. 

https://github.com/t8or/recequip-wiki-scraper/blob/master/data_to_import.csv

So if the page you want has one, feel free to make a PR editing that csv

**Comment by Brandon1212b** [comment]
I just added 11 more to that file

**Comment by BkRandii** [comment]
Can we add revenants if at all possible please

**Comment by Brandon1212b** [comment]
Sure man. I just added it to my pr waiting for approval
